doc/usecases/layers/paths_layer.md

```diff
@@ -0,0 +1,23 @@
+# Use Case: Paths Layer
+
+## Summary
+
+- **Scope:** Paths Layer
+- **Level:** User Goal
+- **Actors:** App User
+- **Brief:** The user can add, edit, move and delete path elements in their plan using the paths layer to represent the movement and flow of traffic in their garden
+- **Status:** Draft
+
+## Scenarios
+
+- **Precondition:**
+  The user has opened the app and has selected the paths layer.
+- **Main success scenario:**
+  The user successfully adds, edits, moves and deletes path elements in their plan using the paths layer. This includes positioning the path elements in the appropriate location.
```

Comment:
See above. Also thickness of paths should be select-able (stepping stones, small way, ... roads)